packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.1
+    tag: v1.0.3-rc2
```

Comment:
You can use the other issue you created (that tracks the new chart version) to remember to tag the real image so that the RC2 image gets replaced.

Reply:
@cbron if we update the image tag later, can we then keep the appVersion same as v1.0.3

Reply:
yes